src/app/pages/signing/signing.component.ts

```diff
@@ -8,9 +8,44 @@ import { Component, OnInit } from '@angular/core';
 
 export class SigningComponent implements OnInit {
   public signingBoxText = '';
+  // fonts and colors
+  public currentFont = 'Arial';
+  public backgroundColor = '#ffffff';
+  public textColor = '#000000';
+
+  selectFont(font: string) {
+    this.currentFont = font;
+  }
+  makeBold(textbox: HTMLInputElement) {
+    const start = textbox.selectionStart;
+    const end = textbox.selectionEnd;
+    const text = textbox.value;
+    this.signingBoxText = text.slice(0, start) + '**' + text.slice(start, end) + '**' + text.slice(end);
+    textbox.select();
+  }
+  makeItalics(textbox: HTMLInputElement) {
+    const start = textbox.selectionStart;
+    const end = textbox.selectionEnd;
+    const text = textbox.value;
+    this.signingBoxText = text.slice(0, start) + '*' + text.slice(start, end) + '*' + text.slice(end);
+    textbox.select();
+  }
+  makeUnderline(textbox: HTMLInputElement) {
+    const start = textbox.selectionStart;
+    const end = textbox.selectionEnd;
+    const text = textbox.value;
+    this.signingBoxText = text.slice(0, start) + '<u>' + text.slice(start, end) + '</u>' + text.slice(end);
+    textbox.select();
+  }
```

Comment:
Add blank line after function
```suggestion
  }
  
```